firstvoices/backend/serializers/song_serializers.py

```diff
@@ -36,11 +36,13 @@ class SongSerializer(
     visibility = WritableVisibilityField(required=True)
 
     title_translation = serializers.CharField(
-        required=False, allow_blank=True, default=""
+        required=False, allow_blank=True, allow_null=True, default=""
+    )
+    introduction = serializers.CharField(
+        required=False, allow_blank=True, allow_null=True, default=""
     )
-    introduction = serializers.CharField(required=False, allow_blank=True, default="")
     introduction_translation = serializers.CharField(
-        required=False, allow_blank=True, default=""
+        required=False, allow_blank=True, allow_null=True, default=""
```

Comment:
What difference does it make to have the default value here? I am wondering whether all these optional CharFields should have the same settings or whether some of them need this extra default value

Reply:
Oh good point! After a bit of local testing I found that there is no effect on this default value changing `None` to `""`. It seems like it would be used to change `""` values to some other string value if `""` was provided by a PUT request. As per: https://www.django-rest-framework.org/api-guide/fields/#core-arguments

For consistency though I can update the other charfields with `default=""` as that does describe the behaviour that occurs. What do you think?